### ISSUE TITLE
Node.js Port / IP Address

### ISSUE THREAD

**Issue by mrtimothyduong** [barrowclift/shelf#1]
Hi Barrowclift,

I'm having issues with the nginx server blocks and not 100% sure what to input for the ipaddress:proxyport for the 'proxy_pass'

Is there a specific Port for the node.js application?

Everything else is working in terms of the node.js apps based on the logs, just not able to publish it.

Regards,

Timothy

**Comment by barrowclift** [comment]
I apologize, Timothy, the code block in the documentation was simply incorrect. I collect documentation as I go, and it looks like I had forgotten to going back and correct that snippet with a working version. Thanks so much for bringing this to my attention, it's [now been updated](https://github.com/barrowclift/shelf/commit/f963f5671bc09f14f40c57dda6d8ec6ac4458a74#diff-7d442b7eb49f5fc377f51e74b291cfc1) to be correct.

__To answer your question:__ The `proxy_pass` value should be `http://localhost:10800;`, the value in the original example didn't make any sense and was also pointing to the wrong port entirely (8080, instead of Shelf's default of 10800).

---

Just in case you changed the port defined in `server.js@DEFAULT_SERVER_SETTINGS.port`, then you'll unfortunately need to merge in [f963f](https://github.com/barrowclift/shelf/commit/f963f5671bc09f14f40c57dda6d8ec6ac4458a74) or change it back to the default value of 10800. While correcting the documentation, I noticed there was unfortunately a static usage of the default value still left in the codebase, which has now been updated to correctly use the config value.
